dali/test/python/test_operator_uniform.py

```diff
@@ -27,14 +27,13 @@ def test_uniform_default():
         pipe.set_outputs(dali.fn.uniform(shape=[1e6]))
     pipe.build()
     oo = pipe.run()
-    possibly_uniform_distribution = oo[0].as_array()[0]
+    data = oo[0].as_array()[0]
     # normalize to 0-1 range
-    possibly_uniform_distribution_kstest = (possibly_uniform_distribution + 1) / 2
-    _, pv = st.kstest(rvs=possibly_uniform_distribution_kstest, cdf='uniform')
-    assert pv > 0.05, "`possibly_uniform_distribution` is not a uniform distribution"
-    assert (-1 <= possibly_uniform_distribution).all() and \
-            (possibly_uniform_distribution < 1).all(), \
-                "Value returned from the op is outside of requested range"
+    data_kstest = (data + 1) / 2
+    _, pv = st.kstest(rvs=data_kstest, cdf='uniform')
+    assert pv > 0.03, "`data` is not a uniform distribution"
```

Comment:
I have mixed filling about this. 0.05 is usually a confidence threshold, if we lower this we may miss the actual error.
Maybe we should change the number of samples and repeat the kstest multiple times?

Reply:
I would agree with Janusz on this. Moreover, I believe it should be rather `assert pv >= 0.5`, my thinking is as follows:

As I see in `kstest` docs:
> Under the null hypothesis, the two distributions are identical, F(x)=G(x).

Thus in out case, if the null hypothesis holds, we can assume that `data_kstest` is a normal distribution. If the p-value is small, we can reject the null hypothesis, meaning **for sure** `data_kstest` is not a normal distribution. This part is I believe obvious.

Now, from this point different sources tell different things. E.g. (wikipedia)[https://en.wikipedia.org/wiki/P-value] suggests, that: If the p-value is not small, we can't reject the null hypothesis, meaning we do not know if the `data_kstest` is a normal distribution.

OTOH, (random article found in the web)[https://towardsdatascience.com/an-intuitive-explanation-of-hypothesis-testing-and-p-values-8391db2c90f] states, that `0.05` might be assumed as the `true/false` threshold. The author of this one accepts the null hypothesis when `pv >= 0.05`.

I'd more trust wikipedia than the second guy. However, I believe, that we do not need to be 100% mathematically correct here, so having `pvalue` large enough might do in our case. What does it mean that `pvalue` is large enough? I'd say 0.5, but it might be 0.4 as well, or any other number here (e.g. 0.1...) ;)